api/src/main/java/org/pmiops/workbench/api/CloudTaskInitialCreditsExhaustionController.java

```diff
@@ -105,6 +109,9 @@ private void handleExhaustedUsers(Set<DbUser> newlyExhaustedUsers) {
           initialCreditsService.updateInitialCreditsExhaustion(user, true);
           // delete apps and runtimes
           deleteAppsAndRuntimesInInitialCreditsWorkspaces(user);
+          // Unlink billing account in VWB
+          // This is done to stop extra charges from being incurred
+          vwbUserService.unlinkBillingAccountForUserPod(user);
```

Comment:
we need a feature flag for the whole VWB initil credit. otherwise, this code will hit prod once merged. 
And having a feature flag would be easier for us to control release once VWB launch ticket is approved

Reply:
yes I'll add one